style.css

```diff
@@ -0,0 +1,111 @@
+body {
+    font-family: 'Roboto', sans-serif, arial, 'Slabo 27px', serif, "Benton Sans", "Helvetica Neue", helvetica;
+    margin: 5em;
+    font-size: 22px;
+    line-height: 1em ; 
+    background-color:aliceblue;
+  }
+
+  a {
+    outline: none;
+    text-decoration: none;
+    padding: 2px 1px 0;
+    color: #0065CC;
+  }
+
+  #profilepic {
```

Comment:
Even though an id makes more sense for something you're using once, I recommend using a class for all styling. It will prevent issues in the future with CSS selector specificity.